mxnet/buildspec.yml

```diff
@@ -37,31 +37,31 @@ context:
       target: deep_learning_container.py
 
 images:
-  # BuildMXNetCPUTrainPy3DockerImage:
-  #   <<: *TRAINING_REPOSITORY
-  #   build: &MXNET_CPU_TRAINING_PY3 false
-  #   image_size_baseline: 2253
-  #   device_type: &DEVICE_TYPE cpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py37
-  #   os_version: &OS_VERSION ubuntu16.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
-  #   context:
-  #     <<: *TRAINING_CONTEXT
-  # BuildMXNetGPUTrainPy3DockerImage:
-  #   <<: *TRAINING_REPOSITORY
-  #   build: &MXNET_GPU_TRAINING_PY3 false
-  #   image_size_baseline: 7159
-  #   device_type: &DEVICE_TYPE gpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py37
-  #   cuda_version: &CUDA_VERSION cu110
-  #   os_version: &OS_VERSION ubuntu16.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
-  #   context:
-  #     <<: *TRAINING_CONTEXT
+  BuildMXNetCPUTrainPy3DockerImage:
+    <<: *TRAINING_REPOSITORY
+    build: &MXNET_CPU_TRAINING_PY3 false
+    image_size_baseline: 2253
+    device_type: &DEVICE_TYPE cpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py37
+    os_version: &OS_VERSION ubuntu16.04
+    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
+    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+    context:
+      <<: *TRAINING_CONTEXT
+  BuildMXNetGPUTrainPy3DockerImage:
+    <<: *TRAINING_REPOSITORY
+    build: &MXNET_GPU_TRAINING_PY3 false
+    image_size_baseline: 7159
+    device_type: &DEVICE_TYPE gpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py37
+    cuda_version: &CUDA_VERSION cu110
+    os_version: &OS_VERSION ubuntu16.04
```

Comment:
update os_version everywhere to ubuntu18.04

Reply:
done